### DISCUSSION TITLE
i3status Bar Still Visible in when i3lock is running

### DISCUSSION THREAD

**Discussion by jdholtz** [i3/i3#5763] [Q&A] [answered]
Hey everyone. I've recently had some weird bugs with my lock screen (specifically that my screen isn't showing the screensaver after my lock delay until I press a key or move the mouse). However, that may not be an issue with i3, but when I trigger the lock screen (`i3lock -c 000000`), I can still see the i3status bar when the screensaver is up by pressing the Mod key (my bar is hidden by default). Here is my bar config:
```i3config
bar {
        font pango:Terminus 11
        status_command i3status -c ~/.config/i3/bar_config
        position top
        mode hide
        hidden_state hide
        modifier Mod4
}
```
and here is the display output for `inxi -Gxx`:
```
Display: server: X.Org v: 21.1.9 with: Xwayland v: 23.2.2
    compositor: xcompmgr driver: X: loaded: modesetting unloaded: vesa
    alternate: fbdev,intel dri: iris gpu: i915 display-ID: :0 screens: 1
```
_Note: I tried without Xwayland and still get the issue_

How can I fix this so the bar is not visible by pressing the Mod key in the i3lock screen? Thanks! Let me know if I can provide any more information.

**Comment by jdholtz** [comment]
It ended up being an issue with xcompmgr. I guess the solution is to just not use xcompmgr, but is there a better solution than disabling xcompmgr?

The reason I am using xcompmgr is to fix annotating not working in Zoom -- https://www.reddit.com/r/i3wm/comments/n3wzaj/fix_for_black_screen_in_zoom_when_annotating/

**Reply by orestisfl** [reply, marked as answer]
Try picom and also check out https://github.com/i3/i3lock/issues/347 and https://github.com/i3/i3lock/issues/333, specifically the workaround: https://github.com/i3/i3lock/issues/333#issuecomment-1329881590

**Reply by jdholtz** [reply]
The workaround worked. Thanks very much!
